src/ansible_builder/user_definition.py

```diff
@@ -245,6 +245,15 @@ def validate(self):
                     self.builder_image = ImageDescription(images, 'builder_image')
                     self.build_arg_defaults['EE_BUILDER_IMAGE'] = self.builder_image.name
 
+            if (
+                self.version >= 3
+                and self.build_arg_defaults["EE_BASE_IMAGE"] == constants.build_arg_defaults["EE_BASE_IMAGE"]
+            ):
+                logging.warning(
+                    "Using the outdated base image '%s' might "
+                    "result in the build failures.", self.build_arg_defaults['EE_BASE_IMAGE']
```

Comment:
```suggestion
                    "result in build failures.", self.build_arg_defaults['EE_BASE_IMAGE']
```

Reply:
Downstream AAP packaged builder patches the value for `EE_BASE_IMAGE` in `contants.py`. We likely don't want this warning appearing in that case. Maybe we should just hardcode the quay image string here?

Reply:
Done.